doc/api/stream.md

```diff
@@ -825,7 +831,33 @@ result in increased throughput.
 
 If both `'readable'` and [`'data'`][]  are used at the same time, `'readable'`
 takes precedence in controlling the flow, i.e. `'data'` will be emitted
-only when [`stream.read()`][stream-read] is called.
+only when [`stream.read()`][stream-read] is called. But when last `'readable'`
+was removed stream will try to [`stream.resume()`][stream-resume] to fulfill
+`'data'` listeners.
+
+```js
+const fs = require('fs');
+const rr = fs.createReadStream('bar.txt');
+rr.once('readable', () => {
+  console.log('readable');
+});
+rr.on('data', (chunk) => {
+  console.log(chunk.toString());
```

Comment:
I think we might want to add `rr.setEncoding('utf8')`, or pass the `encoding: 'utf8'` option to `createReadStream` rather than using `.toString()`, to encourage best practices

(The current code could fail if `bar.txt` contains UTF-8 characters outside the ASCII range)